src/smc-webapp/project_files.cjsx

```diff
@@ -377,24 +377,40 @@ NoFiles = rclass
         </Button>
 
     # TODO: Make better help text
-    render_help_text : ->
+    render_help_alert : ->
         last_folder_index = @props.file_search?.lastIndexOf('/')
         # Non-empty search and there is a file divisor ('/')
         if @props.file_search?.length > 0 and last_folder_index > 0
-            # Ends with a '/' ie. only folders
-            if last_folder_index == @props.file_search.length - 1
-                if last_folder_index isnt @props.file_search.indexOf('/')
-                    # More than one sub folder
-                    text = "Path #{@props.file_search} will be ensured to exist."
-                else
-                    # Only one folder
-                    text = "Creates a folder named #{@props.file_search}"
-            else
-                text = "#{@full_path_text().slice(last_folder_index + 1)} will be created under the folder path #{@props.file_search.slice(0, last_folder_index + 1)}"
             <Alert style={marginTop: '10px'} bsStyle='info'>
-                {text}
+                {@render_help_text(last_folder_index)}
             </Alert>
 
+    render_help_text : (last_folder_index) ->
```

Comment:
There has to be a better way to render arbitrary styles in the middle of text but google results are only giving me React Native examples.